changelog/unreleased/bugfixes/7061.md

```diff
@@ -0,0 +1 @@
+- Fixed Native crash signal 11 (SIGSEGV), code 1 (SEGV_MAPERR) that may happen at libmapbox-common.so at `SqlitePersistentStorage#createQuery`.
```

Comment:
happened

Reply:
I'm merging as is to not wait for CI

Reply:
It's more to identify the crash. If the user sees that crash in their report, they understand that this particular crash is fixed in this version. While there can be lots of Native crash signal 11 (SIGSEGV), code 1 (SEGV_MAPERR).